src/tm_devices/commands/gen_e3e9uu_lpdmso/bus.py

```diff
@@ -7518,10 +7548,6 @@ def threshold(self) -> BusBItemParallelClocksourceThreshold:
             - BUS:B<x>:PARallel:CLOCKSOUrce:THReshold <NR3>
             - BUS:B<x>:PARallel:CLOCKSOUrce:THReshold?
             ```
-
-        Info:
```

Comment:
Please go through this entire file and make sure all the info isn't getting deleted.